go.mod

```diff
@@ -86,7 +86,10 @@ require (
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
 	github.com/golang/mock v1.6.0 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
-	github.com/google/go-cmp v0.5.8 // indirect
+	github.com/google/go-cmp v0.5.9 // indirect
+	github.com/google/go-github/v28 v28.1.1 // indirect
```

Comment:
can you run `go mod tidy` please to remove unused dependencies

Reply:
fixed